js-miniapp-sdk/test/miniapp.spec.ts

```diff
@@ -21,20 +21,21 @@ const window: any = {};
 (global as any).window = window;
 
 window.MiniAppBridge = {
-  getUniqueId: sandbox.stub(),
-  requestPermission: sandbox.stub(),
-  requestCustomPermissions: sandbox.stub(),
-  loadInterstitialAd: sandbox.stub(),
-  loadRewardedAd: sandbox.stub(),
-  showInterstitialAd: sandbox.stub(),
-  showRewardedAd: sandbox.stub(),
-  shareInfo: sandbox.stub(),
-  getPlatform: sandbox.stub(),
-  getUserName: sandbox.stub(),
-  getProfilePhoto: sandbox.stub(),
-  getContacts: sandbox.stub(),
-  getAccessToken: sandbox.stub(),
-  setScreenOrientation: sandbox.stub(),
+  getUniqueId: sinon.stub(),
```

Comment:
These should be using `sandbox.stub` rather than `sinon.stub`. This ensures that the stubs are reset before each test.